### ISSUE TITLE
[Security] Rogue swsssdk package on PyPI

### ISSUE THREAD

**Issue by bluecmd** [sonic-net/sonic-buildimage#9825]
#### Description

I was running some pytests and accidentally the swsssdk package from https://pypi.org/project/swsssdk/ was pulled.
I noticed and thought it was weird that it existed on pypi and looked at it. It looks like it is a security research
project where the only thing it does is pinging a server that it has been installed.

```python3
class CustomInstall(install):
    def run(self):
        install.run(self)
        x = requests.get("https://en16bbemuui9jdp.m.pipedream.net")
        print(x)
```

However, nothing is stopping the owner with replacing this package with something hostile.

#### Steps to reproduce the issue:
1. Run `python3 src/sonic-py-common/setup.py test`  **[POTENTIALLY DANGEROUS]**

#### Describe the results you received:

I didn't save the terminal output but the result was that the swsssdk was downloaded from PyPI instead of erroring out telling me that I have to install it manually.

#### Describe the results you expected:

Some libraries are required to be manually installed for testing - they should never be tried to fetched from an online source.

**Comment by qiluo-msft** [comment]
Just to clarify, it is an issue in the dev environment, but does not appears in the build system?

**Comment by bluecmd** [comment]
> Just to clarify, it is an issue in the dev environment, but does not appears in the build system?

Yes, correct. This is only a problem if a developer tries to execute the tests outside the Docker build environment.

**Comment by qiluo-msft** [comment]
Since it is not a build system issue, closiong this issue. However any contribution to improve the dev experience is welcome.
